modules/cli/pages/podman.adoc

```diff
@@ -52,6 +52,15 @@ For example, `dnf install podman-plugins` for RPM based distributions.
 
 NOTE: CNI will be deprecated in the future in preference of Netavark.
 --
+* Podman service endpoint.
++
+--
+Use `systemctl status podman.socket` to make sure the Podman API Socket is running.
+
+Use `systemctl --user enable --now podman.socket` to start the  Podman API Socket.
```

Comment:
@pwright, this looks better than what I proposed.  It would be good to have @fgiorgetti's input on it, just in case.